qa/tasks/cephfs/mount.py

```diff
@@ -169,6 +169,26 @@ def assert_and_log_minimum_mount_details(self):
                      get_file(self.client_remote, self.client_keyring_path,
                               sudo=True).decode())
 
+    def is_stuck(self):
+        """
+        Check if mount is stuck/in a hanged state.
+        """
+        if not self.is_mounted():
+            return False
+
+        retval = self.client_remote.run(args=f'sudo stat {self.hostfs_mntpt}',
+                                        omit_sudo=False, wait=False).returncode
+        if retval == 0:
+            return False
+
+        time.sleep(10)
```

Comment:
@batrick 

Current code works fine but it needs to wait for for few seconds to be fully sure that operation on the mount point has hanged. Waiting at every call to CephFSMount.umount()/CephFSMount.umount_wait() is a bit too much. I thought checking for process state code in output of `ps -el ` might work but unfortunately it didn't show that the process has hanged (i.e. is in a uninterruptible sleep). Any ideas on how to do it without waiting?